libs/l10n/l10n.ts

```diff
@@ -0,0 +1,30 @@
+type Locale =
+  | "en-US"
+  | "es"
+  | "fr"
+  | "ja"
+  | "ko"
+  | "pt-BR"
+  | "ru"
+  | "zh-CN"
+  | "zh-TW";
+type TranslatedLocale = Exclude<Locale, "en-US">;
+type Strings = {
+  "en-US": string;
+} & Record<TranslatedLocale, string>;
+
+function localString(strings: Strings) {
```

Comment:
Should this be `localeString` rather than `localString`?

```suggestion
function localeString(strings: LocaleString) {
```

Reply:
Fixed in [3ae5d78](https://github.com/mdn/yari/pull/11011/commits/3ae5d784a5ec37711f9d45687b603b9ed4da2ed8).